frontend/app/src/components/home/PollContent.svelte

```diff
@@ -24,13 +24,16 @@
 
     $: numberOfVotes = totalVotes(content);
 
+    $: cannotVote =
+        content.ended || readonly || (haveIVoted && !content.config.allowUserToChangeVote);
```

Comment:
If the config is set to allow multiple then they can still vote on other options.

Reply:
yes I thought it made sense that this new setting only applied for single vote polls

Reply:
Oh wait - I see what you mean

Reply:
although - if allow multiple is true, allow user to change will also always be true